src/core_modules/capture-core/components/Pages/MainPage/WorkingListsCommon/hooks/useWorkingListsCommonStateManagement.js

```diff
@@ -187,13 +185,16 @@ const useWorkingListsContext = (dispatch: ReduxDispatch, { storeId }: { storeId:
         lastTransaction,
     }), shallowEqual);
 
-    const loadedContext = useSelector(({ workingListsContext }) => workingListsContext[storeId]);
+    const { listDataRefreshTimestamp, lastTransactionOnListDataRefresh, ...loadedContext } =
+        useSelector(({ workingListsContext }) => workingListsContext[storeId]) || {};
```

Comment:
Why we want to get the `listDataRefreshTimestamp` and `lastTransactionOnListDataRefresh` out of the `loadedContext` object?